temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowClientCallsInterceptor.java

```diff
@@ -61,6 +62,8 @@ public interface WorkflowClientCallsInterceptor {
 
   WorkflowSignalWithStartOutput signalWithStart(WorkflowSignalWithStartInput input);
 
+  <R> WorkflowUpdateWithStartOutput<R> updateWithStart(WorkflowUpdateWithStartInput<R> input);
```

Comment:
Can we add javadocs here?

Reply:
We should probably also call out if you are overriding `start` you may want to override `updateWithStart` like we do for `signalWithStart `

Reply:
@Quinn-With-Two-Ns regarding javadocs, anything in particular you'd expect? None of the other ones have real javadocs other than referring to what else you might want to override.

Reply:
I would say something like

```
Intercepts update with start calls from {@link WorkflowStub#updateWithStart} and {@link WorkflowClient#updateWithStart}
```

How does that sounds

Reply:
Done! https://github.com/temporalio/sdk-java/pull/2199/commits/e77581535609824811e76cb36227daa515531ce5